models/issue.go

```diff
@@ -1374,7 +1373,8 @@ func parseCountResult(results []map[string][]byte) int64 {
 		return 0
 	}
 	for _, result := range results[0] {
-		return com.StrTo(string(result)).MustInt64()
+		c, _ := strconv.ParseInt(string(result), 10, 64)
+		return c
```

Comment:
Why even loop here?

Reply:
No idear ... its old code

Reply:
unrelated to this pull ... but I'll refactor it

Reply:
@lafriks do you know a better way to get the value of the first (unknown) key of a map?

Reply:
Oh, it's map, than no I don't 😅